course2/index.js

```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+app.use(express.static(path.join(__dirname, 'public')));
+
+app.listen(3005, () => console.log('Example app listening on port 3000!'));
```

Comment:
If you change the port it's good to also change the comment. It took me a minute to figure out why the page didn't load :) 
On Windows you can also run `start npm start` to start the process in a new cmd window which gets killed when closed. That is if you had problems with "zombie" processes.

Reply:
Oh, sorry about that. I had to change the port because I was on another PC with 3000 taken.